.ci_support/migrations/cuda120.yaml

```diff
@@ -62,13 +66,13 @@ cuda_compiler_version:         # [linux64 and os.environ.get("CF_CUDA_ENABLED",
   - 12.0                       # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
 c_compiler_version:            # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12                         # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 11                         # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
 cxx_compiler_version:          # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12                         # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 11                         # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
 
 fortran_compiler_version:      # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 12                         # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 11                         # [linux64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
Filed issue ( https://github.com/conda-forge/pp-sketchlib-feedstock/issues/43 ) to track reverting this once the upstream issue ( https://github.com/pybind/pybind11/issues/4606 ) is resolved